### DISCUSSION TITLE
Need Help

### DISCUSSION THREAD

**Discussion by 8Bite-play** [gildor2/UEViewer#285] [Q&A]
Hi, I tried to open Thief game files and I keep getting this error.
![Umodel_Error](https://user-images.githubusercontent.com/73986391/197383023-ddaf8d11-abef-4983-9c98-17eaa2f4d574.jpg)
Can you please help me? I am new to this.

**Comment by gildor2** [comment]
Animations aren't supported for this game (see the compatibility table). You should disable them.

**Reply by 8Bite-play** [reply]
OK, Thank you so much!
